Package.swift

```diff
@@ -104,18 +103,11 @@ var package = Package(
     .target(
       name: "ServerRouter",
       dependencies: [
-        "ServerRoutes",
         "SharedModels",
         .product(name: "ApplicativeRouter", package: "Web"),
         .product(name: "Tagged", package: "swift-tagged"),
       ]
     ),
-    .target(
-      name: "ServerRoutes",
-      dependencies: [
-        "SharedModels"
-      ]
-    ),
     .testTarget(
       name: "ServerRouterTests",
```

Comment:
should we move this to `SharedModels` too?

Reply:
Oh, these are the _router_ tests, not tests for the route type.

Reply:
oh ha! yeah duh